charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/namespace-azure-disk-csi.yaml

```diff
@@ -0,0 +1,11 @@
+---
+{
+  "apiVersion": "v1",
+  "kind": "Namespace",
+  "metadata": {
+    "name": "azure-disk-csi",
```

Comment:
what is this namespace for? I could see v2 driver are still in the kube-system ns.

Reply:
The is the namespace for our custom resources.